spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -905,6 +906,21 @@ public void setEnabled(boolean enabled) {
 		 */
 		private Integer selectors = -1;
 
+		/**
+		 * Maximum amount of threads.
+		 */
+		private Integer maxThreads = 200;
```

Comment:
would not be better to set the maximum number of threads based on the number of available cores?

Reply:
The values here are the default of the embedded servers. We don't deviate from that unless necessary. If you want to pursue this idea, I'd raise an issue against the Jetty project.